app/src/main/java/com/imdbmovieapp/data/local/dao/FavoriteMovieDao.kt

```diff
@@ -0,0 +1,21 @@
+package com.imdbmovieapp.data.local.dao
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.imdbmovieapp.data.local.entity.FavoriteMovieEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface FavoriteMovieDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertMovie(favoriteMovieEntity: FavoriteMovieEntity)
+
+    @Delete
+    suspend fun deleteMovie(favoriteMovieEntity: FavoriteMovieEntity)
+
+    @Query("SELECT * FROM 'movieTable'")
```

Comment:
You can eliminate `' '` from queries because they don't make a difference and it can cause some problems in other SQLs (even though it is not a problem in `SQLite`)